webapp/channels/src/components/file_preview_modal/file_preview_modal_main_actions/file_preview_modal_main_actions.tsx

```diff
@@ -96,17 +91,13 @@ const FilePreviewModalMainActions: React.FC<Props> = (props: Props) => {
         });
     }
     const publicLink = (
-        <OverlayTrigger
-            delayShow={Constants.OVERLAY_TIME_DELAY}
+        <WithTooltip
+            id='link-variant-icon-tooltip.text'
             key='filePreviewPublicLink'
             placement={tooltipPlacement}
+            title={publicTooltipMessage}
             shouldUpdatePosition={true}
             onExit={() => setPublicLinkCopied(false)}
```

Comment:
Can you verify what is the effect of this and what would be the problem if we were to remove this line?